src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -34,14 +34,17 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        String email = authentication.getPrincipal().toString();
```

Comment:
```suggestion
        String email = authentication.getName();
```
the same for other similar cases